.github/workflows/github-nightly-windows.yaml

```diff
@@ -0,0 +1,54 @@
+name: Nightly, Hosted Runners, Windows
+
+on:
+  schedule:
+    # 9:50 AM UTC (~2:50 AM Mountain), Monday-Friday
+    - cron: "50 9 * * 1-5"
+  workflow_dispatch:
```

Comment:
Is that file duplicating the continuous workflow or does it actually have a different definition besides the triggering "on" section?